webpack.config.ts

```diff
@@ -612,6 +613,12 @@ if (
       },
     };
     appConfig.output!.publicPath = '/_static/dist/sentry/';
+
+    appConfig.plugins?.push(
+      new WebpackHookPlugin({
+        onBuildStart: ['yarn run sidecar'],
```

Comment:
What does this do?

Reply:
Starts a node http server on 8969 until you shutdown the process

Reply:
In the future would it make sense to rename this binary to `spotlight-sidecar`. Feels pretty ambiguous what this is right now.

Reply:
https://github.com/getsentry/spotlight/issues/165

Reply:
It's starts the sidecar (proxy) for Spotlight